### DISCUSSION TITLE
Cards' images are not optimized

### DISCUSSION THREAD

**Discussion by luba-media** [gatsbyjs/gatsby#29755] [Help]
On my site, https://www.learnmoreaboutyourself.com/, the cards' images are not converted from png to webp like in regular article pages.
Any idea how to fix this?

Thanks in advance!

**Comment by ascorbic** [comment]
Can you share the code?

**Reply by luba-media** [reply]
@ascorbic I've added you to this private repository.
I'm new to gatsby and github, so not sure whether that's the most appropriate way to go.

Thanks again for this.

**Reply by luba-media** [reply]
@ascorbic the name of the repo is luba-media/learnmoreaboutyourself.
Can you please let me know whether you can take a look?
Many thanks!
